src/common/controllers/ads-controller.js

```diff
@@ -116,11 +134,16 @@ class AdsController extends FakeEventTarget implements IAdsController {
       this._playAdBreak({
         position: this._player.currentTime || 0,
         ads: adPod,
-        played: false
+        played: false,
+        loadedPromise: Promise.resolve()
```

Comment:
maybe add an option to pass prebid config also here as this api could be used directly from kaltura player